src/core/modules/Planet.cpp

```diff
@@ -682,43 +683,53 @@ QString Planet::getInfoString(const StelCore* core, const InfoStringGroup& flags
 			{
 				// For compute the Moon age we use geocentric coordinates
 				QString moonPhase = "";
-				double eclJDE = earth->getRotObliquity(core->getJDE());
-				double ra_equ, dec_equ, lambdaMoon, lambdaSun, beta;
-				StelCore* core1 = StelApp::getInstance().getCore();
-				bool state = core1->getUseTopocentricCoordinates();
+				StelCore* core1 = StelApp::getInstance().getCore(); // FIXME: Why do we need a different reference to the (singular) core here?
```

Comment:
New reference to core need to avoid problem with nomenclature shifts

Reply:
> 
> 
> Code for Moon phase looks weird now - please check it again

The idea is: mark "sharp" phases, but also name the phases in between for much longer time. But in fact, I just see it still can be improved...

Reply:
I think this FIXME maybe deleted or changed to NOTE.

Reply:
I think core1 is not needed. It points to the same thing as core.

Reply:
> I think core1 is not needed. It points to the same thing as core.

No problem, but please resolve problem of jumping Moon and lunar nomenclature in this case